R/randomForest_tidiers.R

```diff
@@ -479,7 +586,6 @@ augment.randomForest.classification <- function(x, data = NULL, newdata = NULL,
 
     # When na.omit is used, case-wise model attributes will only be calculated
     # for complete cases in the original data. All columns returned with
-    # augment() must be expanded to the length of the full data, inserting NA
```

Comment:
What is this removal of comment about?

Reply:
Sorry this is a mistake. I'll review the comments in this code.